server/src/main/java/org/apache/iotdb/db/service/thrift/impl/TSServiceImpl.java

```diff
@@ -1762,7 +1762,9 @@ public TSStatus createAlignedTimeseries(TSCreateAlignedTimeseriesReq req) {
               dataTypes,
               encodings,
               compressors,
-              req.measurementAlias);
+              req.measurementAlias,
+              req.tagsList,
+              req.attributesList);
```

Comment:
Please consider the compatibility with the interface of old version. May add a new interface to support tag/attribute is a better solution.

Reply:
Hi, `tagsList` and `attributesList` are Optional in rpc.thrift...

Is there any compatibility problem about it?

Reply:
If so, it will be ok.